### ISSUE TITLE
button pressed color?

### ISSUE THREAD

**Issue by MobileMon** [urbn/URBNAlert#60]
How to set button touch down color?

**Comment by ryang1428** [comment]
Currently this is not implemented. What would need to happen is creating styler properties for `buttonBackgroundHighlightedColor`, `destructionButtonBackgroundHighlightedColor`, and `cancelButtonBackgroundHighlightedColor`. If I have some time in the next few days I can add them

**Comment by ryang1428** [comment]
You could technically achieve this in the current release, but its not as clean. You could create an instance of a URBNAlertAction, then do something like:

```
URBNAlertAction *action = [URBNAlertAction actionWithTitle.....
[action.actionButton setBackgroundImage:[self imageWithColor:[UIColor redColor]] forState:UIControlStateHighlighted];
```

Note the imageWithColor function, you'll need a similar method like the one from here: http://stackoverflow.com/questions/14523348/how-to-change-the-background-color-of-a-uibutton-while-its-highlighted

**Comment by MobileMon** [comment]
Yes certainly there is multiple ways to handle it either with images or pressed states. I've used this extension before and it worked out well:

https://github.com/GabrielMassana/ButtonBackgroundColor-iOS

Just would be nice to have it already bundled up in URBN

**Comment by ryang1428** [comment]
Addressed in v1.9.5 https://github.com/urbn/URBNAlert/pull/64
